### PR TITLE
Request new match

### LINKED ISSUE #42: Request new match without having to retake the test
It would be cool if you could get a new match without having to retake the entire quizz

### PR DESCRIPTION
**Pull request by q-Sci**
Fix #42

### CODE REVIEW

File: questions.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// all questions used in the political test
+
+const questions = [
```
Comment (GregTCLTK) [inline questions.ts:3]: ig we could just store the questions in a questions.json file

**Comment by GregTCLTK** [comment]
this pr is kinda critical as it probably breaks all other open and future prs due to massive merge conflicts

**Comment by johan-t** [comment]
> this pr is kinda critical as it probably breaks all other open and future prs due to massive merge conflicts

I agree, I think we should first merge #71 and then merge this one

**Comment by johan-t** [comment]
Can we also merge this directly into main?

**Comment by q-Sci** [comment]
> Can we also merge this directly into main?

Yes we can. But what happened to the concept of the pre-production branch?

**Comment by johan-t** [comment]
> > Can we also merge this directly into main?
> 
> Yes we can. But what happened to the concept of the pre-production branch?

As long as this branch is a branch of the development branch it is fine, we just need to remember to merge main into development before starting something new

**Review by johan-t — COMMENTED** [review]
General Notes: 
- I am not getting a new match, only the one I have been matched with before

File: commands/match/match-command.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    // if not, an error hint is displayed
+    if (userResponses.length === questions.length) {
+        await interaction.reply({
+            content: 'Neues Match wird ermittelt. Bitte schaue in deinen Direktnachrichten nach :)',
+            ephemeral: true,
+        });
+        sendQuestion(interaction);
+    } else {
+        await interaction.reply({
+            content: 'Bitte beantworte den Meinungstest vollständig, bevor du mit Anderen gematcht werden kannst! Bitte nutze dazu den Befehl `/test`.',
```
Comment (johan-t) [inline commands/match/match-command.ts:92]: Could we maybe automatically trigger the test command if the user has not finished the test?

**Review by johan-t — CHANGES_REQUESTED** [review]

**Comment by johan-t** [comment]
Also could you think of a way where we could put a button that starts the /match command to make it easier for users to find this feature?

**Comment by GregTCLTK** [comment]
please don't use feature branches to completely re-structure the entire codebase in the future

**Pull request merged**